connect/api/src/main/java/org/apache/kafka/connect/data/ConnectSchema.java

```diff
@@ -220,7 +225,9 @@ public static void validateValue(Schema schema, Object value) {
                 expectedClasses = SCHEMA_TYPE_CLASSES.get(schema.type());
 
         if (expectedClasses == null)
-            throw new DataException("Invalid Java object for schema type " + schema.type() + ": " + value.getClass());
+            throw new DataException("Invalid Java object for schema type " + schema.type()
```

Comment:
The error message is updated to say "required field". Do we actually know that this is a required field here?